governance/gatekeeper/gatekeeper_yaml_table.adoc

```diff
@@ -0,0 +1,260 @@
+[#gatekeeper-operator-sample-{gk-tag}]
+== {gk-operator} custom resource
+
+[source,yaml]
+----
+apiVersion: operator.gatekeeper.sh/v1alpha1
+kind: Gatekeeper
+metadata:
+  name: gatekeeper
+spec:
+  audit:
+    auditInterval: 60s
+    constraintViolationLimit: 20
+    auditFromCache: Disabled
+    auditChunkSize: 500
+    emitAuditEvents: Disabled
+ifeval::[{gk-y-ver} >= 14]
+    auditEventsInvolvedNamespace: Disabled <.>
+endif::[]
```

Comment:
If the version attributes are going away, this conditional will no longer work. Will these attributes be included here? If not, we'll have to figure out how to distinguish things added in a particular version (like maybe the note attached to it can start with `Added in v3.14` or `>=v3.14`?)